test/benchmarks/driver_bench/src/driver.mts

```diff
@@ -161,6 +163,8 @@ export function metrics(test_name: string, result: number, tags?: string[]): Met
         ])
       )
     },
+    created_at: new Date().toISOString(),
```

Comment:
are created_at and completed_at intentionally the same? do we know what those are usually supposed to signify for the perf metrics?

Reply:
The AC in NODE-6837 says: "Ensure that the time a benchmark begins and concludes are correctly recorded and reported in the info field of the metrics that get written out"

Reply:
> They are used for search-ability, for example, if we want to pull the latest performance results (before they are processed into a timeseries) they aren't really used to determine anything downstream

Quoting Ahmad here. So seems like it doesn't relate to our usage, if we were using DSI it would be set for DSI's purposes

Reply:
> it is fine since you probably aren't going to be querying the raw data directly anyways

more quoting confidence. I think we can remove that from the AC (same for the BSON one)

Reply:
ok then that should be fine